website/pages/en/quick-start.mdx

```diff
@@ -35,133 +30,141 @@ Using [yarn](https://yarnpkg.com/):
-- Add another contract(optional): you can add another contract
+- ABI: If the ABI is not auto-populated, you will need to input it manually as a JSON file
+- Start Block: You should input the start block to optimize subgraph indexing of blockchain data. Locate the start block by finding the block where your contract was deployed.
+- Contract Name: Input the name of your contract
+- Index contract events as entities: It is suggested that you set this to true, as it will automatically add mappings to your subgraph for every emitted event
+- Add another contract(optional): You can add another contract.
 
 See the following screenshot for an example for what to expect when initializing your subgraph:
 
 ![Subgraph command](/img/subgraph-init-example.png)
 
-## 4. Write your subgraph
+### 4. Write your subgraph
 
-The previous commands create a scaffold subgraph that you can use as a starting point for building your subgraph. When making changes to the subgraph, you will mainly work with three files:
+The `init` command in the previous step creates a scaffold subgraph that you can use as a starting point to build your subgraph.
 
-- Manifest (`subgraph.yaml`) - The manifest defines what datasources your subgraphs will index.
-- Schema (`schema.graphql`) - The GraphQL schema defines what data you wish to retrieve from the subgraph.
-- AssemblyScript Mappings (`mapping.ts`) - This is the code that translates data from your datasources to the entities defined in the schema.
+When making changes to the subgraph, you will mainly work with three files:
 
-For more information on how to write your subgraph, see [Creating a Subgraph](/developing/creating-a-subgraph).
+- Manifest (`subgraph.yaml`)- defines what datasources your subgraphs will index.
+- Schema (`schema. graphql`) - defines what data you wish to retrieve from the subgraph.
+- AssemblyScript Mappings (`mapping.ts`) - translates data from your datasources to the entities defined in the schema.
 
-## 5. Deploy to Subgraph Studio
+For a detailed breakdown on how to write your subgraph, check out [Creating a Subgraph](/developing/creating-a-subgraph/).
 
-Once your subgraph is written, run the following commands:
+### 5. Deploy to your subgraph
 
-```sh
-$ graph codegen
-$ graph build
-```
+Remember, deploying is not the same as publishing.
 
-- Authenticate and deploy your subgraph. The deploy key can be found on the Subgraph page in Subgraph Studio.
+- When you deploy a subgraph, you push it to [Subgraph Studio](https://thegraph.com/studio/), where you can test, stage and review it.
+- When you publish a subgraph, you are publishing it onchain to the decentralized network.
 
-```sh
-$ graph auth --studio <DEPLOY_KEY>
-$ graph deploy --studio <SUBGRAPH_SLUG>
-```
+1. Once your subgraph is written, run the following commands:
+
+   ```sh
+   $ graph codegen
+   $ graph build
+   ```
```

Comment:
```suggestion
   ```sh
   graph codegen
   graph build
   ```
```
(otherwise, `$` is also copied if the user clicks on the Copy button)